docs/apis-clients/tasklist-api/overview.md

```diff
@@ -10,6 +12,8 @@ In this document, we'll go over the basics on how to consume the Tasklist GraphQ
 
 Tasklist provides a GraphQL API at endpoint `/graphql`.
 
+In Camunda 8 that would be `https://${REGION}.tasklist.camunda.io:443/${CLUSTER_ID}/graphql`
```

Comment:
I think something along these lines is a bit more clear because it doesn't just point to Camunda 8

```suggestion
From Camunda 8 onwards the endpoint `https://${REGION}.tasklist.camunda.io:443/${CLUSTER_ID}/graphql`
```

Reply:
Go ahead and spell out "Camunda Platform 8" as well.